### PR TITLE
Addition of Buildpsec file and .ebextension for elastic beanstalk

### PR DESCRIPTION
**Pull request by whos4n3**
### What

Describe what you have changed and why.

### How to review

Describe the steps required to test the changes.

### CODE REVIEW

File: .ebextensions/sshd.config [hunk 1]
```diff
@@ -0,0 +1,21 @@
+packages:
+  yum:
+    python27-devel: []
```
Comment (VivianAllen) [inline .ebextensions/sshd.config]: is this python 2.7x reference significant?

File: buildspec.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.8
```
Comment (VivianAllen) [inline buildspec.yml]: I'm pretty sure the container is running 3.6x - don't know if that is gonna cause any issues?

File: .ebextensions/sshd.config (hunk 1 above)
Comment (whos4n3) [reply .ebextensions/sshd.config]: It's Elastic beanstalk's default configuration. Not needed anymore.

**Comment by whos4n3** [comment]
Not required as the proposed changes were for Elastic beanstalk but now Architecture has been moved to elastic container service.

**Pull request closed without merging**